src/store/csv/stub/content/pupils-at-expected-standards/2023-03/.~lock.1.csv#

```diff
@@ -0,0 +1 @@
+,davidbailey,Davids-MBP.broadband,17.01.2024 13:29,file:///Users/davidbailey/Library/Application%20Support/LibreOffice/4;
```

Comment:
I assume this is auto-generated or something, but it needs to be removed.

Reply:
okay